include/drivers/clock_control/stm32_clock_control.h

```diff
@@ -164,11 +164,19 @@
 #define STM32_CSI_FREQ		0
 #endif
 
-#if DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(clk_lsi), fixed_clock, okay)
-#define STM32_LSI_ENABLED	1
+/* the STM32_LSI_FREQ is given by the LSI or LSI1 RC oscillator, even if not enabled */
+#if DT_PROP(DT_NODELABEL(clk_lsi), clock_frequency)
 #define STM32_LSI_FREQ		DT_PROP(DT_NODELABEL(clk_lsi), clock_frequency)
+#elif DT_PROP(DT_NODELABEL(clk_lsi1), clock_frequency)
```

Comment:
Instead of adding LSI1, we can tweak wb.dtsi to set 'lsi' as an alias to `lsi1` in the following way:
```
      lsi1: lsi@address {
          bla;
          foo;
      };
```

Reply:
is it 
```
		clk_lsi: clk-lsi {
			#clock-cells = <0>;
			compatible = "st,stm32-lsi-clock";
			clock-frequency = <DT_FREQ_K(32)>;
			status = "disabled";
		};
```
in the dts/arm/st/wb/stm32wb.dtsi

Reply:
defining the lsi1 of the stm32wb as lsi node in its DTS:
```
clk_lsi: clk-lsi1 {
...
}
```
So there is only one reference  **DT_NODELABEL(clk_lsi)**   in the driver